src/address/legacy-address.ts

```diff
@@ -0,0 +1,12 @@
+export default interface LegacyAddress {
```

Comment:
what's the difference between internal and legacy address? is this maybe `PaymentLegacyAddress` ?

Reply:
Legacy address is the format `/checkout.php` expects.